package.json

```diff
@@ -36,8 +36,8 @@
     "dotenv": "5.0.1",
     "form-data": "2.2.0",
     "lodash": "4.17.10",
-    "node-fetch": "1.7.1",
-    "zapier-platform-schema": "6.1.0"
+    "node-fetch": "2.1.2",
```

Comment:
is this is a safe change to make? Seems like there are some actual [incompatibilities](https://github.com/bitinn/node-fetch/blob/master/UPGRADE-GUIDE.md) here, though I don't know if we're using any of these one way or the other.

Reply:
Looking around the [changelog](https://github.com/bitinn/node-fetch/blob/master/CHANGELOG.md), I don't see any 🔥using `2.1.2`. This part of [v2.0.0 general changes](https://github.com/bitinn/node-fetch/blob/master/CHANGELOG.md#general-changes) pushed me to bump it though:

> Enhance: start testing on Node.js v4.x, v6.x, v8.x LTS, as well as v9.x stable

Reply:
To be on a safe side, let's stick with `1.7.1` in this release since it isn't broken running on Node.js `8.10.0`.